### PR TITLE
12 delete /api/comments/comment

### PR DESCRIPTION
**Pull request by stephen-jonesy**
Models have been moved to separate files and error-handling controllers are now in the new file errors.js.

### CODE REVIEW

**Review by EddTheDuck — APPROVED** [review]
Looks good

File: __tests__/app.test.js [hunk 1]
```diff
@@ -249,3 +249,30 @@ describe('GET /api/users', () => {
         })
     });
 });
+
+describe('DELETE /api/comments/:comment_id', () => {
+    it('returns status of 204 when sent the correct id and comment is removed from the comments table', () => {
+        return request(app)
```
Comment (Rosem10) [inline __tests__/app.test.js:345]: Please also check that the comment has actually been deleted from the database by making a request to your GET commentsById endpoint and asserting that you expect the response to be 404 comment not found after doing the patch.

File: controllers/articlesController.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const { selectArticles, selectArticleById, patchArticleVotes } = require("../models/articles.model");
+
+exports.getArticles = (req, res, next) => {
+    
+    return selectArticles()
+    .then(({rows}) => {
```
Comment (Rosem10) [inline controllers/articlesController.js]: It's best practice to de-construct the rows in the model before returning the result up again into the controller.

**Review by Rosem10 — APPROVED** [review]
Looking good - just one test to add please!

**Pull request merged**